sympy/solvers/tests/test_solveset.py

```diff
@@ -32,6 +33,27 @@
 n = Symbol('n', real=True)
 
 
+def test_dummy():
```

Comment:
That was temporary and should not have stayed in.
Thanks for spotting this.
(Anything that is not a change in an import line should never have made it into the PR.)